hurma/spark_export_jobs/write_csv_to_binary.py

```diff
@@ -82,24 +85,45 @@
             'updated_at',
             'category_groups_list'
         ]
-    },
-    'organizations': {
+    }
+
+
+ORGANIZATIONS_DATA_CONFIGURATION = {
         'sep': ',',
         'fields': [
             'uuid',
             'name',
             'permalink',
         ]
-    },
-    'parent_organizations': {
+    }
+
+PARENT_ORGANIZATIONS_DATA_CONFIGURATION = {
         'sep': ',',
         'fields': [
             'uuid',
             'parent_uuid',
         ]
-    },
+    }
+
+
+DATETIME_UNIFICATION_REQUIRED_DATA_TYPE = 'funds'
```

Comment:
minor: it is preferable to use set/tuple an "if data_type in DATETIME_UNIFICATION_REQUIRED_DATA_TYPE"
this will be extendable only by adding a new type into a set